ambassador/tests/t_circuitbreaker.py

```diff
@@ -61,8 +61,9 @@ def manifests(self) -> str:
 """
 
         return self.format(RBAC_CLUSTER_SCOPE + AMBASSADOR, image=os.environ["AMBASSADOR_DOCKER_IMAGE"],
-                           envs=envs, extra_ports="", capabilities_block="") + STATSD_MANIFEST.format(name='cbstatsd-sink',
-                                                                                                      target=self.__class__.TARGET_CLUSTER)
+                           envs=envs, extra_ports="", capabilities_block="") + \
+               STATSD_MANIFEST.format(name='cbstatsd-sink', image=self.test_image['stats'],
+                                      target=self.__class__.TARGET_CLUSTER)
```

Comment:
Is `image=self.test_image['stats']` & `{image}` clearer than `self=self` & `{self.test_image[stats]}` that's used elsewhere?  Consistency is nice.

Reply:
Right now, there are separate `manifest`s for the statsd things that are handled totally differently from way most `manifest`s are handled. That whole thing needs to be untangled (almost certainly by ripping it out and replacing it with the standard mechanism) but I don't want to lump that in here. 😐